terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -2,4 +2,10 @@ environment = "cidev"
 aws_profile = "development-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
```

Comment:
If the log_level is defined in the [S3](https://github.com/companieshouse/ecs-service-configs-dev/blob/main/development-eu-west-2/cidev/company-accounts.api.ch.gov.uk.env) .env file, the log_level from these profile/vars can be removed.